server/model/json.go

```diff
@@ -71,6 +71,29 @@ func (a Assertion) MarshalJSON() ([]byte, error) {
 	})
 }
 
+func (na *NamedAssertions) UnmarshalJSON(data []byte) error {
+	type encodedNamedAssertions struct {
+		Name       string
+		Assertions []Assertion
+	}
+
+	var namedAssertion encodedNamedAssertions
+	if err := json.Unmarshal(data, &namedAssertion); err != nil {
```

Comment:
do we need to keep this old code? I know it's nice, but it's also a maintenance overhead, and we are not in a stable point, so I think it's easier to clean the very few DBs that uses this than to keep this soon-to-rot code

Reply:
I think we should at least make patch versions retro-compatible. The way I look at our versioning right now is that we our `minor` versions are our beta major versions. 

I'm thinking about removing this as soon as we get to v0.8